### PR TITLE
Sending local images through chat

### PR DESCRIPTION
**Pull request by ghost**
I completed the functionality that allow users to send local images through the chat. I added constraints in the jsp code to only accept image files when the browse option pops up to avoid any errors for both profile and chat. Also I added the profile link to all the nav bars that way users can now go to the profile from pretty much all pages. Please let me know any suggestions.

### CODE REVIEW

File: src/main/java/codeu/helper/UploadChat.java [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+    /**
+     * Sets the UserStore used by this servlet. This function provides a common setup method for use
+     * by the test framework or the servlet's init() function.
+     */
+    void setUserStore(UserStore userStore) {
+        this.userStore = userStore;
+    }
+
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response)
```
Comment (KevinWorkman) [inline src/main/java/codeu/helper/UploadChat.java]: Is this function actually used?

**Review by KevinWorkman — APPROVED** [review]
It would be cool to see some test classes for new code, but I'm not going to be super strict about it. It's up to you what you think is interesting to explore.

This looks good to me overall, other than the one comment I left below.

File: src/main/java/codeu/helper/UploadChat.java (hunk 1 above)
Comment (ghost) [reply src/main/java/codeu/helper/UploadChat.java]: Not really. I removed it

**Review by mgomez03 — APPROVED** [review]
Wow, good job Jose!

**Pull request merged**